autoload/iced/message.vim

```diff
@@ -65,5 +65,22 @@ function! iced#message#error(k, ...) abort
   call iced#message#error_str(msg)
 endfunction
 
+function! iced#message#short(k, ...) abort
+  let msg = iced#message#get(a:k)
+  if !empty(a:000)
+    let msg = call('printf', [msg] + a:000)
+  endif
```

Comment:
Calling `printf` would not be needed in `iced#util`.
One utility function should do only one operation.